src/records.py

```diff
@@ -0,0 +1,57 @@
+import aiohttp
+from heartbeat import *
+
+
+class Records:
+    def __init__(self, job_type, task_type):
+        self.job_type = job_type
+        self.task_type = task_type
+
+    async def get(self, base_url, job_id):
```

Comment:
get is not being used what is it's purpose?

Reply:
fixed in 
559c61a